weld/EAP7-839_CDI_2_0_support.adoc

```diff
@@ -0,0 +1,52 @@
+= EAP7-839 CDI 2.0 Support
+:author:            Matej Novotny
+:email:             manovotn@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org
+
+== Overview
+
+This issue has two parts.
+Firstly it brings Weld 3.x/CDI 2.0 into Wildfly.
+Secondly, it introduces support for switch between CDI 1.2 (EE7) and CDI 2.0 (EE 8) for the sake of certification.
+
+== Issue Metadata
+
+=== Issue:
+
+* {issue-base-url}/EAP7-839[EAP7-839]
+
+=== Related Issues:
+
+* {issue-base-url}/WFLY-6465[WFLY-6465]
+* {issue-base-url}/WFLY-9269[WFLY-9269]
+* {issue-base-url}/WFLY-9768[WFLY-9768]
+
+=== Dev Contacts:
+
+* mailto:{email}[{author}]
+
+=== QE Contacts:
+
+* mailto:{email}[{author}]
```

Comment:
This should be you @manovotn, right?

Reply:
Unless you want to join in as well? ;-)
Nice catch, will update

Reply:
This is actually correct, those are fields from the beginning of the document and it will render my name and email (sorry it was probably too late when I wrote last comment) - it will look like [this](https://github.com/manovotn/wildfly-proposals/blob/EAP7-839/weld/EAP7-839_CDI_2_0_support.adoc)